comms/src/transports/tcp.rs

```diff
@@ -108,10 +107,10 @@ impl TcpTransport {
 impl Default for TcpTransport {
     fn default() -> Self {
         Self {
-            recv_buffer_size: None,
-            send_buffer_size: None,
+            // recv_buffer_size: None,
+            // send_buffer_size: None,
             ttl: None,
-            keepalive: None,
+            // keepalive: None,
```

Comment:
remove?

Reply:
This (and all the others in this file) could come back - tokio doesn't support these socket options at the moment. I'd rather leave them in